<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDocProperty</class>
 <widget class="QDialog" name="EditDocProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="FLayoutInput">
     <item row="1" column="0">
      <widget class="QLabel" name="LabelMachine">
       <property name="text">
        <string>Machine:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LabelDevStep">
       <property name="text">
        <string>Dev. step:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="EditDevStep">
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="placeholderText">
        <string>100, 150, 300, ...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LabelFromSN">
       <property name="text">
        <string>From SN:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="EditFromSN">
       <property name="maxLength">
        <number>11</number>
       </property>
       <property name="placeholderText">
        <string>12345/67890</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="LabelToSN">
       <property name="text">
        <string>To SN:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="EditToSN">
       <property name="placeholderText">
        <string>67890</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="ComboMachine">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="LabelDocument">
       <property name="text">
        <string>Document:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="LabelFilename">
       <property name="text">
        <string>&lt;filename&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <widget class="QPushButton" name="ButtonOpenFile">
       <property name="text">
        <string>Open file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOpenFileDir">
       <property name="text">
        <string>Open file directory</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="HSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
